src/healthkit.cs

```diff
@@ -2442,4 +2557,220 @@ interface HKWorkoutRouteQuery {
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface HKQuantitySeriesSampleBuilder
+	{
+		[Export ("initWithHealthStore:quantityType:startDate:device:")]
+		IntPtr Constructor (HKHealthStore healthStore, HKQuantityType quantityType, NSDate startDate, [NullAllowed] HKDevice device);
+
+		[Export ("quantityType", ArgumentSemantic.Copy)]
+		HKQuantityType QuantityType { get; }
+
+		[Export ("startDate", ArgumentSemantic.Copy)]
+		NSDate StartDate { get; }
+
+		[NullAllowed, Export ("device", ArgumentSemantic.Copy)]
+		HKDevice Device { get; }
+
+		[Export ("insertQuantity:date:error:")]
+		bool Insert (HKQuantity quantity, NSDate date, [NullAllowed] out NSError error);
+
+		[Async, Internal]
+		[Export ("finishSeriesWithMetadata:completion:")]
+		void FinishSeries ([NullAllowed] NSDictionary metadata, Action<NSArray<HKQuantitySample>, NSError> completionHandler);
+
+		[Async]
+		[Wrap ("FinishSeries (metadata.Dictionary, completionHandler)")]
+		void FinishSeries ([NullAllowed] HKMetadata metadata, Action<NSArray<HKQuantitySample>, NSError> completionHandler);
+
+		[Export ("discard")]
+		void Discard ();
+	}
+
+	[Watch (5,0), NoiOS]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface HKLiveWorkoutDataSource
+	{
+		[Export ("typesToCollect", ArgumentSemantic.Copy)]
+		NSSet<HKQuantityType> TypesToCollect { get; }
+
+		[Export ("initWithHealthStore:workoutConfiguration:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (HKHealthStore healthStore, [NullAllowed] HKWorkoutConfiguration configuration);
+
+		[Export ("enableCollectionForType:predicate:")]
+		void EnableCollection (HKQuantityType quantityType, [NullAllowed] NSPredicate predicate);
+
+		[Export ("disableCollectionForType:")]
+		void DisableCollection (HKQuantityType quantityType);
+	}
+
+	[NoWatch, iOS (12,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface HKFHIRResource : NSSecureCoding, NSCopying
+	{
+		[Internal]
+		[Export ("resourceType")]
+		NSString _ResourceType { get; }
+
+		HKFHIRResourceType ResourceType { [Wrap ("HKFHIRResourceTypeExtensions.GetValue (_ResourceType)")] get; }
+
+		[Export ("identifier")]
+		string Identifier { get; }
+
+		[Export ("data", ArgumentSemantic.Copy)]
+		NSData Data { get; }
+
+		[NullAllowed, Export ("sourceURL", ArgumentSemantic.Copy)]
+		NSUrl SourceUrl { get; }
+	}
+
+	[Watch (5,0), iOS (12,0)]
+	[DisableDefaultCtor]
+	[BaseType (typeof(HKQuantitySample))]
+	interface HKCumulativeQuantitySeriesSample
+	{
+		[Export ("sum", ArgumentSemantic.Copy)]
+		HKQuantity Sum { get; }
+	}
+
+	[NoWatch, iOS (12,0)]
+	[DisableDefaultCtor]
+	[BaseType (typeof(HKSample))]
+	interface HKClinicalRecord : NSSecureCoding, NSCopying
+	{
+		[Export ("clinicalType", ArgumentSemantic.Copy)]
+		HKClinicalType ClinicalType { get; }
+
+		[Export ("displayName")]
+		string DisplayName { get; }
+
+		[NullAllowed, Export ("FHIRResource", ArgumentSemantic.Copy)]
+		HKFHIRResource FHIRResource { get; }
```

Comment:
Unsure if we should turn this into `FhirResource `

Reply:
Same opinion, not sure which one would be better.

Reply:
Changed them in the next commit after looking in the entire file.